src/Momento.Sdk/Config/Transport/StaticTransportStrategy.cs

```diff
@@ -27,13 +34,27 @@ public class StaticGrpcConfiguration : IGrpcConfiguration
     /// <param name="grpcChannelOptions">Customizations to low-level gRPC channel configuration</param>
     /// <param name="minNumGrpcChannels">minimum number of gRPC channels to open</param>
     /// <param name="socketsHttpHandlerOptions">Customizations to the SocketsHttpHandler</param>
-    public StaticGrpcConfiguration(TimeSpan deadline, GrpcChannelOptions? grpcChannelOptions = null, int minNumGrpcChannels = 1, SocketsHttpHandlerOptions? socketsHttpHandlerOptions = null)
+    /// <param name="keepAlivePingTimeout">Override the time to wait for a response from a keepalive ping</param>
+    /// <param name="keepAlivePingDelay">Override how often the client sends keepalive pings the server</param>
+    /// <param name="keepAlivePermitWithoutCalls">Sets option to send keepalive pings from the client without any outstanding RPCs</param>
+    public StaticGrpcConfiguration(
+        TimeSpan deadline, 
+        GrpcChannelOptions? grpcChannelOptions = null, 
+        int minNumGrpcChannels = 1, 
+        SocketsHttpHandlerOptions? socketsHttpHandlerOptions = null,
+        TimeSpan? keepAlivePingTimeout = null, 
+        TimeSpan? keepAlivePingDelay = null, 
+        bool? keepAlivePermitWithoutCalls = null
+    )
     {
         Utils.ArgumentStrictlyPositive(deadline, nameof(deadline));
         this.Deadline = deadline;
         this.MinNumGrpcChannels = minNumGrpcChannels;
-        this.GrpcChannelOptions = grpcChannelOptions ?? new GrpcChannelOptions();
```

Comment:
I think maybe what you want to do is make this `?? DefaultGrpcChannelOptions()`, and then define that function instead of having a `FromGrpcConfig` function? It's the same issue I mentioned earlier where this SDK is a little different because we are using the .NET types directly. So we shouldn't need to create our own config types to model anything that can be expressed inside of the .NET GrpcChannelOptions type, because we're already exposing it.